### ISSUE TITLE
Adding two lines to GdxDefinition.gwt.xml makes GWT builds much smaller

### ISSUE THREAD

**Issue by GrantOjanen** [libgdx/libgdx#5536]
#### Issue details
By default an simple game build for HTML is larger than 10mb, but this can be lowered to about 2.5mb.

I added the `set-property` and `collapse-property` tags at the end. The game is a lot smaller now on HTML, and I tried it out on Firefox, Chromium, Safari (Mac), Edge, and IE11. The game also takes a minute or two shorter to compile to HTML! It would be nice if this was added to the default project generated with gdx-setup.jar.

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit trunk//EN" "http://www.gwtproject.org/doctype/2.8.0/gwt-module.dtd">
<module rename-to="html">
	<inherits name='com.badlogic.gdx.backends.gdx_backends_gwt' />
	<inherits name='com.badlogic.gdx.controllers.controllers-gwt' />

	<inherits name='MyGdxGame' />

	<entry-point class='com.grantojanen.RainDrop.client.HtmlLauncher' />
	<set-configuration-property name='xsiframe.failIfScriptTag' value='FALSE'/>
	<set-configuration-property name="gdx.assetpath" value="../android/assets" />
	<set-property name="user.agent" value="gecko1_8, safari"/>
	<collapse-property name="user.agent" values="*" />
</module>
```

#### Version of LibGDX and/or relevant dependencies
libGDX 1.9.9
(I'm using Android Studio)

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [x] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by MrStahlfelge** [comment]
That's a good find. If I am understanding [this file](https://github.com/gwtproject/gwt/blob/master/user/src/com/google/gwt/useragent/UserAgent.gwt.xml) correct, the ommited user agents are ie8, ie9, ie10. Should be safe to use for a game!

**Comment by GrantOjanen** [comment]
> That's a good find. If I am understanding [this file](https://github.com/gwtproject/gwt/blob/master/user/src/com/google/gwt/useragent/UserAgent.gwt.xml) correct, the ommited user agents are ie8, ie9, ie10. Should be safe to use for a game!

Ya! IE10 doesn't even support webGL anyways.

The `collapse-property` tag merges gecko1_8, and safari. I don't know if that could have some negative side effects.

**Comment by MrStahlfelge** [comment]
Can confirm statement now. Compile time went down from 5 minutes to 2 minutes for my game, compressed directory size from 7 MB to 4 MB. Thanks again!

**Comment by tommyettinger** [comment]
About a 2x speedup here on a project that I think has a lot of dead code to be eliminated. It goes from 1 min 57 seconds on a clean build without these 2 lines (21.9 MB once symbol maps are removed from `WEB-INF/deploy/html/symbolMaps`), to 59 seconds on a clean build with those 2 lines (11.0 MB dist size once symbol maps are removed). Those dist folders aren't compressed. With zip compression, the no-fix folder is 9.5 MB, and the yes-fix folder is 7.1 MB. My machine is a slightly-worn 6th-gen i7 quad laptop with DDR4 RAM; Java and GWT are installed on an SSD but the project is on an HDD, and the machine runs Windows 7.
